<launch>
    <node pkg="ethernet_test" type="ethernet_node" name="ethernet_node" output="screen">
        <param name="device_ip" type="string" value="127.0.0.1" />
        <param name="device_port" type="int" value="8888" />
        <param name="mcu_ip" type="string" value="127.0.0.1" />
        <param name="mcu_port" type="int" value="8889" />
        <param name="debug" type="bool" value="true" />
        <param name="poll_timeout" type="int" value="2000" />
    </node>
</launch>